CHANGELOG.md

```diff
@@ -5,6 +5,13 @@
 * [#436](https://github.com/stripe/stripe-php/pull/436) Introduce new `Idempotency` error class
 
 ## 6.0.0 - 2018-02-07
+Major version release. List of backwards incompatible changes to watch out for:
++ The minimum PHP version is now 5.4.0. If you're using PHP 5.3 or older, consider upgrading to a more recent version.
+* `\Stripe\AttachedObject` no longer exists. Attributes that used to be instances if `\Stripe\AttachedObject` (such as `metadata`) are now instances of `\Stripe\StripeObject`.
```

Comment:
One more small one: "that used to be instance if" should be "of" instead of "if" :)

Reply:
🤦‍♂️  Fixed.